src/Generation/GapicClientGenerator.php

```diff
@@ -755,15 +755,17 @@ private function rpcMethod(MethodDetails $method): PhpClassMember
                 PhpDoc::example($this->examples()->rpcMethodExample($method), PhpDoc::text('Sample code:')),
                 $isStreamedRequest
                     ? null
-                    : Vector::zip($method->requiredFields, $required, fn ($field, $param) =>
+                    : Vector::zip(
+                        $method->requiredFields->filter(fn ($f) => !$f->isOneOf || $f->isFirstFieldInOneof()),
+                        $required,
+                        fn ($field, $param) =>
                         PhpDoc::param(
                             $param,
                             PhpDoc::preFormattedText(
                                 !$field->isOneOf
                                     ? $field->docLines->concat($docExtra($field))
-                                    : Vector::new(['Maps to the required proto oneof '
-                                        . $field->containingMessage->getOneofDecl()[$field->oneOfIndex]->getName()
-                                        . '.'
+                                    : Vector::new([
+                                        'Maps to the required proto oneof ' . $field->getOneofDesc()->getName() . '.'
```

Comment:
As it's a wrapper class now, could you change this to something else that is not "Maps"?
I think it's misleading to code readers.

Reply:
Done.